src/tasks/CodeDeployDeployApplication/task.json

```diff
@@ -111,6 +111,25 @@
             "visibleRule": "deploymentRevisionSource = s3",
             "helpMarkDown": "The Amazon S3 object key of the previously uploaded archive file containing the deployment revision artifacts."
         },
+        {
+            "name": "filesAcl",
+            "type": "pickList",
+            "label": "Access Control (ACL)",
+            "defaultValue": "none",
+            "required": false,
+            "visibleRule": "deploymentRevisionSource = workspace",
+            "helpMarkDown": "The canned Access Control List (ACL) to apply to the uploaded content. See [Canned ACL](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl) for an explanation of the possible values. By default, no ACL is applied.",
+            "options": {
+                "none": "none",
+                "private": "private",
+                "public-read": "public read",
+                "public-read-write": "public read write",
+                "authenticated-read": "authenticated read",
+                "aws-exec-read": "aws-exec-read",
+                "bucket-owner-read": "bucket-owner-read",
+                "bucket-owner-full-control": "bucket-owner-full-control"
```

Comment:
Style nit: why do some have hyphens and why do some not? Should we make these values more presentable, e.g. proper caps?

Reply:
Was nabbed from the S3 upload task. It's probably reasonable to change both for consistency.

Reply:
Gotcha, yeah, wouldn't mind seeing everything consistent...it's just kinda jarring to see the two different styles. If you would like to get this in sooner rather than later, feel free to merge as-is and make the change later if you decide on a consistent style you'd like to keep.

Reply:
Went ahead and just changed both since it's such a small change and only affects UI.